# Request
float64 x
float64 y
float64 z

---

# Response
bool success
string message